platforms/common/i2c_spi_buses.cpp

```diff
@@ -508,14 +508,15 @@ int I2CSPIDriverBase::module_start(const BusCLIArguments &cli, BusInstanceIterat
 	}
 
 	bool started = false;
+	bool already_running = false;
 
 	while (iterator.next()) {
 		if (iterator.instance()) {
 			PX4_WARN("Already running on bus %i", iterator.bus());
```

Comment:
add an if here, to avoid bogus warning when running several sensors on the same SPI bus, only separated by CS